src/System/Relay/createRelaySSREnvironment.ts

```diff
@@ -99,7 +102,8 @@ export function createRelaySSREnvironment(config: Config = {}) {
       headers: req => {
         // Determine if the request is cacheable based on the opt-in `@cacheable` directive.
         // If it is, we don't want to send the user's access token even if they are logged in.
-        const isCacheable = isRequestCacheable(req)
+        const isCacheable =
```

Comment:
I'm down to iterate more on this, although I think this is fine for now.

At first I tried to wrap this condition in the `isRequestCacheable` method, but the `req` object we're generally passing around here, as these are all Relay middleware/helpers, is a `RelayRequest` (and not any Express/node request object) - which _doesnt_ have any URL/query params present.